src/api/routers/user_tasks.py

```diff
@@ -1,75 +1,69 @@
     UserTasksAndShiftResponse,
 )
-from src.core.db.models import UserTask
 from src.core.services.shift_service import ShiftService
-from src.core.services.user_task_service import UserTaskService, get_user_task_service
+from src.core.services.user_task_service import UserTaskService
 from src.core.settings import settings
 
 router = APIRouter(prefix="/user_tasks", tags=["user_tasks"])
 
 
-STR_ENTITY_NOT_EXIST = "Задачи с указанным id не существует!"
-SHIFT_NOT_FOUND = "Такая смена не найдена, проверьте id."
-
-
 @router.get(
     "/{user_task_id}",
     response_model=UserTaskResponse,
     response_model_exclude_none=True,
+    status_code=HTTPStatus.OK,
     summary="Получить информацию об отчёте участника.",
     response_description="Полная информация об отчёте участника.",
 )
 async def get_user_report(
     user_task_id: UUID,
-    user_task_service: UserTaskService = Depends(get_user_task_service),
-) -> UserTask:
+    user_task_service: UserTaskService = Depends(),
+) -> dict:
     """Вернуть отчет участника.
 
+    - **user_id**:номер участника
     - **user_task_id**: номер задачи, назначенной участнику на день смены (генерируется рандомно при старте смены)
     - **task_id**: номер задачи
     - **day_number**: номер дня смены
     - **status**: статус задачи
     - **photo_url**: url фото выполненной задачи
     """
-    user_task = await user_task_service.get_or_none(user_task_id)
-    if user_task is None:
-        raise HTTPException(status_code=HTTPStatus.NOT_FOUND, detail=STR_ENTITY_NOT_EXIST)
+    user_task = await user_task_service.get_user_task_with_photo_url(user_task_id)
     return user_task
 
 
 @router.patch(
     "/{user_task_id}",
     response_model=UserTaskResponse,
     response_model_exclude_none=True,
+    status_code=HTTPStatus.OK,
     summary="Изменить статус участника.",
     response_description="Полная информация об отчёте участника.",
 )
-async def change_user_report_status(
+async def update_status_report(
     user_task_id: UUID,
-    request: ChangeStatusRequest,
-    user_task_service: UserTaskService = Depends(get_user_task_service),
-) -> UserTask:
+    update_user_task_status: ChangeStatusRequest,
+    user_task_service: UserTaskService = Depends(),
+) -> dict:
     """Изменить статус отчета участника.
 
+    - **user_id**:номер участника
     - **user_task_id**: номер задачи, назначенной участнику на день смены (генерируется рандомно при старте смены)
     - **task_id**: номер задачи
     - **day_number**: номер дня смены
     - **status**: статус задачи
     - **photo_url**: url фото выполненной задачи
     """
-    user_task = await user_task_service.get_or_none(user_task_id)
-    if user_task is None:
-        raise HTTPException(status_code=HTTPStatus.NOT_FOUND, detail=STR_ENTITY_NOT_EXIST)
-    user_task = await user_task_service.change_status(user_task, request.status)
+    user_task = await user_task_service.update_status(id=user_task_id, update_user_task_status=update_user_task_status)
```

Comment:
Если так будет работать, то именно так и вызывай методы. Код становится короче и красивее.
```suggestion
    user_task = await user_task_service.update_status(user_task_id, update_user_task_status)
```
И добавь пожалуйста в вики рекомендацию о том, как нужно вызывать методы и функции, а потом сделай пост в общем чате!).